filters/united-states/illinois/chicago/public-art-json-to-json.js

```diff
@@ -0,0 +1,36 @@
+const filter = function (data, stringify) {
+    if (typeof data === 'string' || data instanceof String) {
+        data = JSON.parse(data);
+    }
+    
+    let new_data = [];
+
+    data.map(d => {
+        let item = {};
+        item.name = d.art;
+        if (item.name === undefined) {
+            console.log(`Data name not found for art with url ${d.url}`);
+        }
+        let coordinates = { longitude: d.longitude, latitude: d.latitude };
+        if (coordinates.longitude === undefined || coordinates.latitude === undefined) {
+            console.log(`Data coordinates not found for art with url ${d.url}`);
+        }
+
+        let artist = d.artist;
+
+        if(artist !== undefined){
+            item.artist = artist;
```

Comment:
To match the schema, the object generated by the filter must only have the fields `name`, `coordinates`, and `details`. This `artist` field, instead of being at the root of the object, should be nested in another object called `details`.

Eg:
```let details = {}
if (artist !== undefined) {
    details.artist = artist;
}
item.details = details;```